### DISCUSSION TITLE
语音转文字及收费问题

### DISCUSSION THREAD

**Discussion by woshixiaocaomao** [ZuodaoTech/everyone-can-use-english#1211] [Q&A]
情况一：根据我上传的语音已经付费转过一次文字，但是逐句跟读的时候发现有部分语句语音和文字对不上。问题一：为啥会有对不上情况？问题二：重新转为啥还显示需要付费？
情况二：本地上传的英文文档全文翻译是否按次数收费，即翻译一次收一次费，上次翻译过的结果或者记录不留存？

**Comment by an-lee** [comment]
> 问题一：为啥会有对不上情况？

大模型得到的结果不能保证完全正确的，而且每次调用得到的结果也不能保证完全相同。如果只是部分结果不对，可以通过字幕右上角菜单里的“修改”来微调，修改之后会重新对齐，这个过程不涉及调用 AI，不会收费。

> 重新转为啥还显示需要付费？

重新转就是再次调用大模型生成新的结果，需要付费。
（另外如果之前没成功用过本地模型，v0.7.0以后的版本，可以试试本地，可能会成功。用本地模型就不会收费了）

> 上次翻译过的结果或者记录不留存

翻译过的会自动保存在本地。重新进入时点翻译，如果翻译过的，只是显示上次翻译的结果，不会重新收费。想要重新翻译时（觉得翻译质量不行，或者想换一个模型翻译），可以点翻译文本末尾的刷新按钮，会重新翻译，再次收费。
